cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/CredentialsService.kt

```diff
@@ -9,17 +9,18 @@ import com.cognifide.cogboard.config.utils.JsonUtils.getObjectPositionById
 import com.cognifide.cogboard.config.utils.JsonUtils.putIfNotExist
 import com.cognifide.cogboard.config.validation.credentials.CredentialsValidator
 import com.cognifide.cogboard.storage.Storage
+import com.cognifide.cogboard.storage.VolumeStorageFactory
 import io.vertx.core.json.JsonArray
 import io.vertx.core.json.JsonObject
 
 class CredentialsService(
     private val storage: Storage,
-    private val config: JsonObject = storage.loadConfig()
+    private val config: JsonObject = VolumeStorageFactory.credentials().loadConfig()
```

Comment:
we could probably
`import com.cognifide.cogboard.storage.VolumeStorageFactory.credentials`
for better readability

Reply:
changed